evaluation.py

```diff
@@ -0,0 +1,101 @@
+from submodules.HeatMap import HeatMap # https://github.com/LinShanify/HeatMap
+from constants import *
+# from util import *
+
+import matplotlib.pyplot as plt
+import numpy as np
+from PIL import Image
+import cv2
+
+class Evaluation():
+
+    def __init__(self, model, weights, df, num_hg_blocks, batch_size=1):
+        self.model = model              # json of model to be evaluated
+        self.weights = weights          # weights of model to be evaluated
+        self.df = df                    # df of the the annotations we want
+        self.num_hg_blocks = num_hg_blocks
+        self.batch_size = batch_size
+
+    # https://stackoverflow.com/questions/30227466/combine-several-images-horizontally-with-python
+    def hstack_images(self, images, filename):
```

Comment:
**Nitpick**
@nicolepeverley I think you mentioned this is a private method so you could change the name to `__hstack_images`.

Reply:
Good call. I'd avoid `__` prefixes though because double underscores have a special meaning in python. Convention is single underscore `_` to hint private methods/variables

Reply:
> The use of double underscore (__) in front of a name (specifically a method name) is not a convention; it has a specific meaning to the interpreter.

> Python mangles these names and it is used to avoid name clashes with names defined by subclasses.

> This is also called name mangling — the interpreter changes the name of the variable in a way that makes it harder to create collisions when the class is extended later.